### PR TITLE
4K MWC Finals Results

### PR DESCRIPTION
**Pull request by mangomizer**
EPHEMERAL I CAN MAKE PROPER TITLES NOW

### CODE REVIEW

**Comment by Ephemeralis** [comment]
This is 'due' tomorrow, just so you know. Due in the sense that it'd be really weird to release a recap any later than that.

**Comment by mangomizer** [comment]
Don't rush me ahhhhhhhhhh
Should be good for tomorrow.

**Comment by mangomizer** [comment]
I am partly thinking just sacking the Finals recap, and just announcing the results. I pretty much dug myself a hole here not writing the finals recap on time.  Hmm... otherwise I can finish this tuesday.

**Comment by SpartanPlume** [comment]
Couldn't you just do a Finals recap of the 2 weeks in one ?

**Comment by Ephemeralis** [comment]
rename PR to 'results' for my sanity's sake, will look at merging this tomorrow/wed

**Comment by mangomizer** [comment]
Sorry my brain must have farted or something.

File: news/2017-09-21-osu-mania-4k-world-cup-results.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "osu!mania 4K World Cup - Results"
+date: 2017-09-21 00:01:00 +0000
```
Comment (Ephemeralis) [inline news/2017-09-21-osu-mania-4k-world-cup-results.md]: redate for 26 - include rename of filename and associated path for images too

File: news/2017-09-21-osu-mania-4k-world-cup-results.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## <a name="results" id="results"></a>Results
+
+![](/wiki/shared/news/2017-09-21-osu-mania-4k-world-cup-results/podium.jpg)
+
+**With a nearly flawless winning streak, team South Korea are crowned the osu!mania 4K World Cup 2017 Champions!**
+
+Team captain [jakads](https://osu.ppy.sh/users/259972) returns again to lead South Korea towards victory for their second year in a row! Despite being only a 5-man team, their impressive skill and teamwork was what allowed them to prevail. Amazingly, they did not concede a single point at all during the elimination stages. They will proudly carry the title of osu!mania champions for another year!
+
+South Korea takes home the grand prize - $150 per team member, exclusive osu! pins and merchandise, a unique profile badge, and the highly coveted "osu!mania Champion" user title for one year! Many congratulations to them!
```
Comment (Ephemeralis) [inline news/2017-09-21-osu-mania-4k-world-cup-results.md]: add mention that everyone who supported SK with a profile banner now gets the HIGHLY COVETED WINNER'S BANNER until the next tournament starts

**Comment by Ephemeralis** [comment]
tidy and nice and condensed though!

**Pull request merged**